### PR TITLE
frameworks/media: Fix buffer size issue on recording

### PR DESCRIPTION
**Pull request by jarvisjang**
Fix the issue which halt on recording.
Reason: Buffer size unit was different. One is byte, one is frame.

### CODE REVIEW

File: framework/src/media/media_recorder.c [hunk 1]
```diff
@@ -268,7 +268,8 @@ int record_and_write(char *buffer)
 	int size;
 	char inst = ST_PROTO_RECORD_DATA;
 
-	frames = pcm_readi(g_rc.pcmin, buffer, g_rc.buffer_size);
+	frames = pcm_readi(g_rc.pcmin, buffer, pcm_bytes_to_frames(g_rc.pcmin, g_rc.buffer_size));
+	printf("(*)frames: %d\n", frames);
```
Comment (sunghan-chang) [inline framework/src/media/media_recorder.c]: It looks not a mandatory message. How about use debug message?

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/src/media/media_recorder.c (hunk 1 above)
Comment (jarvisjang) [reply framework/src/media/media_recorder.c]: Currently, this is mandatory.
Without printf, recv()/send() functions don't work out well. I guess this is related to the scheduling multi-threads mechanism. After this issue is fixed, I will delete this line.

**Comment by Taejun-Kwon** [comment]
Do not forget remove printf when issue solved.

**Pull request merged**